smartsim/_core/config/redis.conf

```diff
@@ -105,7 +152,7 @@ tcp-backlog 4096
 # incoming connections. There is no default, so Redis will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /tmp/redis.sock
+# unixsocket /run/redis.sock
```

Comment:
Do we use UDS from SmartSim? Might want to align the file here with what we use if so

Reply:
done